Dockerfile.ubuntu

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04 AS build
+FROM ubuntu:16.04 AS build
```

Comment:
Min supported version is now 16 https://learn.microsoft.com/en-us/dotnet/core/whats-new/dotnet-8#minimum-support-baselines-for-linux